finetune.py

```diff
@@ -365,15 +365,13 @@ def finetune(model, train_loader, train_hiddens, args, device, val_loader=None,
         shutil.copy(os.path.join(args.quant_model, "args.pt"), os.path.join(args.save, "args.pt"))
 
     print("\n============ Evaluating perplexity... ============")
-    if args.eval_model_seqlen:
-        quant_model.seqlen = args.eval_model_seqlen
```

Comment:
Rationale: it is best not to rely on a monkey-patched additional property (.seqlen) **that is not originally a part of model**. It seems relatively easy to pass seqlen directly instead.